sourcemap/mapper.go

```diff
@@ -19,23 +19,38 @@ package sourcemap
 
 import (
 	"bufio"
+	"bytes"
 	"strings"
 
 	"github.com/go-sourcemap/sourcemap"
 )
 
-const sourcemapContentSnippetSize = 5
+const (
+	sourcemapContentSnippetSize = 5
+	// TODO: What value do we want here?
+	minMaxLineLength = 16
```

Comment:
I'm assuming 16 is much smaller than we want, it was just convenient for testing.

Reply:
Seems kind of silly if someone limits the output to 1 character, for example. But, I'm also happy to delete it.